api/v3/AwardManager.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @file
+ * Award Manager API file.
+ */
+
+/**
+ * AwardManager.create API specification.
+ *
+ * @param array $spec
+ *   Description of fields supported by this API call.
+ */
+function _civicrm_api3_award_manager_create_spec(array &$spec) {
```

Comment:
No need to add the spec function since we are not adding any pseudo fields. Civi should build the parameters itself.